index.js

```diff
@@ -0,0 +1,21 @@
+/*
+** author: Fulin Shen
+** created on: May 16, 2016
+** description: see above
+*/
+
+var express = require('express');
+var mongoose = require('mongoose');
+var truckRouter = require('./routes/truckRoutes');
+var foodTypeRouter = require('./routes/foodTypeRouter');
+
+var app = express();
+//var trucksDB = mongoose.connect('mongodb://localhost/foodTruckAPI');
```

Comment:
Your mongoose.connect call is commented out and consequently your routes are hanging.